podcasts/AutoplayHelper.swift

```diff
@@ -0,0 +1,53 @@
+import Foundation
+
+class AutoplayHelper {
+    static let shared = AutoplayHelper()
+
+    private let userDefaults: UserDefaults
+    private let userDefaultsKey = "playlist"
+
+    var lastPlaylist: EpisodesDataManager.Playlist? {
+        if let rawDictionary = userDefaults.dictionary(forKey: userDefaultsKey),
+           let dictData = try? JSONSerialization.data(withJSONObject: rawDictionary) {
+            return try? JSONDecoder().decode(EpisodesDataManager.Playlist.self, from: dictData)
+        }
+
+        return nil
+    }
+
+    init(userDefaults: UserDefaults = UserDefaults.standard) {
+        self.userDefaults = userDefaults
+    }
+
+    func playedAt(playlist: EpisodesDataManager.Playlist?) {
```

Comment:
Wdyt about using `playedFrom` or `func played(from playlist: EpisodesDataManager.Playlist?) {` instead of at? 

At first glance the word `at` had my mind thinking this was referring to playback position and not a playlist source. I think switching to `from` could help clarify it.

Reply:
Done in 9c24b5dad5e3510d3636a918f931288ba1031626